assets/cluster-monitoring-operator/prometheus-rule.yaml

```diff
@@ -40,7 +40,9 @@ spec:
         summary: Highly-available workload is incorrectly spread across multiple nodes
           and manual intervention is needed.
       expr: |
-        count by(node,workload,namespace)
+        count without (node)
```

Comment:
This will require updating the runbook since we lost the `node` information in the aggregation so it won't be visible in the alert anymore.

Reply:
the description annotations would break since it's using `$labels.node`

Reply:
Oh indeed, I forgot about that one